ha/cli/exec/clusterExecutor.py

```diff
@@ -15,118 +15,34 @@
 # about this software or licensing, please email opensource@seagate.com or
 # cortx-questions@seagate.com.
 
-import time
-
 from cortx.utils.log import Log
-from ha.execute import SimpleCommand
 from ha import const
 from ha.cli.exec.commandExecutor import CommandExecutor
-from ha.core.error import HAClusterStart
 from ha.core.error import HAUnimplemented
 
+
 class ClusterStartExecutor(CommandExecutor):
 
     def __init__(self):
-
-        # To be removed once the "cortx cluster start" user story [EOS-16248] is started
-        self._execute = SimpleCommand()
+        """
+        Init Cluster Start Executor
+        """
+        super().__init__()
 
     def validate(self) -> str:
-        raise HAUnimplemented("This operation is not implemented.")
-
-    def get_nodes_status(self):
         """
-        Sample output of the const.PCS_STATUS_NODES command
-
-        Pacemaker Nodes:
-         Online: node1 node2
-         Standby:
-         Standby with resource(s) running:
-         Maintenance:
-         Offline:
-        Pacemaker Remote Nodes:
-         Online:
-         Standby:
-         Standby with resource(s) running:
-         Maintenance:
-         Offline:
+        Validate command for cluster start executor
```

Comment:
This must be implemented and should return True/False.
Please check "add node" command implementation.

Reply:
Updated code changes.